packages/wallet/config/webpack.config.js

```diff
@@ -419,16 +419,18 @@ module.exports = function(webpackEnv) {
       ]
     },
     plugins: [
-      new webpack.NormalModuleReplacementPlugin(/.*\/build\/contracts\/.*\.json/, function(
-        resource
-      ) {
-        if (process.env.TARGET_NETWORK !== "development") {
-          resource.request = resource.request.replace(
-            /.*\/build\/contracts/,
-            paths.appPreBuiltContractArtifacts
-          );
-        }
-      }),
+      // TODO: Figure out if we still need this
```

Comment:
@lalexgap I don't think we need to do this. On netlify, we have access to the `@statechannels/nitro-protocol` folder and so we can access the files in there. Assuming `nitro-protocol` also publishes those files in the future, we shouldn't ever need it.